components/director/internal/domain/labeldef/resolver_test.go

```diff
@@ -82,11 +78,11 @@ func TestCreateLabelDefinition(t *testing.T) {
 
 	t.Run("got error on creating Label Definition", func(t *testing.T) {
 		// GIVEN
-		mockPersistanceCtx := &pautomock.PersistenceTxOp{}
-		defer mockPersistanceCtx.AssertExpectations(t)
+		mockPersistenceCtx := &pautomock.PersistenceTxOp{}
+		defer mockPersistenceCtx.AssertExpectations(t)
 		mockTransactioner := &pautomock.Transactioner{}
-		mockTransactioner.On("Begin").Return(mockPersistanceCtx, nil)
-		mockTransactioner.On("RollbackUnlessCommited", mockPersistanceCtx).Return(nil)
+		mockTransactioner.On("Begin").Return(mockPersistenceCtx, nil)
```

Comment:
It would be good to check how many times all mocked methods has been called. For example here, it would be good to add `Once()` expectation.